### PR TITLE
Add PreJSS to Convenience in external-projects

### PR DESCRIPTION
**Pull request by DenisIzmaylov**
Dear,

Thank you for great project.

We would like to share our project and to make SCSS-to-JSS migration more comfortable.

### CODE REVIEW

File: docs/external-projects.md [hunk 1]
```diff
@@ -36,6 +36,8 @@
 
 [jss-css](https://github.com/alexkuz/jss-css) This is a convenience module for JSS that converts CSS string into JSS object. It allows to describe style classes for components in a habitual and little less verbose way.
 
+[PreJSS](https://github.com/axept/prejss) Fast, scoped, component-friendly and fully customizable PostCSS-to-JSS adapter which allow to use plain CSS, PostCSS, SCSS, CSS Modules, Stylus and LESS styles as JSS objects "on-the-fly" though using Literal Template Strings.
```
Comment (kof) [inline docs/external-projects.md]: which allowS
thRough
Its called Tagged Template Strings https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/template_strings#Tagged_Template-Strings

Comment (kof) [inline docs/external-projects.md]: should be just "through Tagged Template Strings" or "using Tagged Template Strings"

**Comment by DenisIzmaylov** [comment]
Oleg, thanks for your feedback. I've updated and squashed commits to keep git history clean.

**Comment by kof** [comment]
merged, thanks! Will be updated on the site with the next version.

**Pull request merged**